test/test_sparse_csr.py

```diff
@@ -3978,12 +3979,13 @@ def test_TensorAsKey(self, device):
         # but key is still valid:
         self.assertEqual(d.get(key5), (key5, 567), **assertEqualOptions)
 
+    @suppress_warnings
```

Comment:
What warnings are being raised that need suppression?

Reply:
https://github.com/pytorch/pytorch/pull/132646/ introduced warnings of the kind
```
UserWarning: bsr_dense_addmm uses non-optimal triton kernel parameters for M=... K=... N=...
```
that is triggered when user is about calling the triton kernel with non-optimal parameters. Here we can suppress these warnings because it is expected that optimal parameters are not available when testing the parameters tuning facilities.